packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -323,6 +370,815 @@ export class FabricTestLedgerV1 implements ITestLedger {
+
+          log.debug("dataConfigTxGen after modificaitons: \n");
+          log.debug(dataConfigTxGen);
+
+          log.debug("dataConfigTxGen after modificaitons: \n");
+          log.debug(dataConfigTxGen);
+
+          const dumpConfigTxGen = yaml.dump(dataConfigTxGen, {
+            flowLevel: -1,
+            quotingType: '"',
+            styles: {
+              "!!int": "decimal",
+              "!!null": "camelcase",
+            },
+          });
+
+          // TODO
+          const regexOrOpening = new RegExp(/OR/g);
+          let replacedConfigTx = dumpConfigTxGen.replace(regexOrOpening, '"OR');
+
+          const regexOrClosing = new RegExp(/\)/g);
+          replacedConfigTx = replacedConfigTx.replace(regexOrClosing, ')"');
+
+          const regexName = new RegExp(/Name:/g);
+          replacedConfigTx = replacedConfigTx.replace(
+            regexName,
+            `&${orgName}\n    Name: `,
+          );
+
+          log.debug(replacedConfigTx);
+
+          // const readersCloseRegEx = new RegExp("')");
+          //dumpConfigTxGen.replace(readersCloseRegEx, "')\"");
+
+          createdFile.filename = `configtx.yaml`;
+          createdFile.filepath = path.join(
+            destinationPath,
+            createdFile.filename,
+          );
+          createdFile.body = replacedConfigTx;
+          log.debug(`Created file at ${createdFile.filepath}`);
+          log.debug(createdFile.filename);
+          await fs.promises.writeFile(createdFile.filepath, createdFile.body);
+
+          return createdFile;
+
+        default:
+          this.log.error(`${fnTag} template type not found`);
+          throw new Error(`${fnTag} template type not found`);
+      }
+    } catch (error) {
+      this.log.error(`populateFile() crashed: `, error);
+      throw new Error(`${fnTag} Unable to run transaction: ${error.message}`);
+    }
+  }
+
+  public setContainer(id: string): void {
+    const fnTag = "FabricTestLedgerV1#setContainer()";
+    let container;
+    try {
+      const docker = new Docker();
+      container = docker.getContainer(id);
+      this.container = container;
+      this.containerId = id;
+    } catch (error) {
+      throw new Error(`${fnTag} cannot set container.`);
+    }
+  }
+
+  public async addExtraOrgs(): Promise<void> {
+    const fnTag = `FabricTestLedger#addOrgX()`;
+    const { log } = this;
+    if (!this.extraOrgs) {
+      throw new Error(`${fnTag}: there are no extra orgs`);
+    }
+
+    log.debug(`
+    Adding ${this.extraOrgs.length} orgs`);
+    for (let i = 0; i < this.extraOrgs.length; i++) {
+      await this.addOrgX(
+        this.extraOrgs[i].path,
+        this.extraOrgs[i].orgName,
+        this.extraOrgs[i].orgChannel,
+        this.extraOrgs[i].certificateAuthority,
+        this.extraOrgs[i].stateDatabase,
+        this.extraOrgs[i].port,
+      );
+    }
+  }
+  // req: AddOrganizationFabricV2Request
+  // returns promise <AddOrganizationFabricV2Response>
```

Comment:
@RafaelAPB The comments do not appear to match the implementation (mismatch in parameters and return type as well). 
I recommend removing the comment to avoid confusing people who would only read the comments through the IDEs documentation viewer feature. Alternatively the documentation could be made up to date but I do not want to put undue extra burden on you with too much nitpicking (I'm doing enough of that already)

```suggestion
```